docs/getting_started.md

```diff
@@ -216,6 +219,92 @@ Here are some resources that can be used to get you started working with `AnnDat
 - [Converting directly from R to Python](https://theislab.github.io/scanpy-in-R/#converting-from-r-to-python)
 - [Homepage for scanpy tutorials](https://scanpy.readthedocs.io/en/latest/tutorials.html)
 
+
+## Special considerations for CITE-seq experiments
+
+
+If you have ADT data from a CITE-seq experiment, you can use the following commands to access raw and normalized ADT expression matrices in the `processed.rds` file object:
```

Comment:
```suggestion
If the dataset that you have downloaded contains samples with ADT data from a CITE-seq experiment, the raw and normalized ADT expression matrices are stored in the `altExp` of the `SingleCellExperiment` object in the `processed.rds` file.
The following commands can be used to access the ADT expression matrices:
```
Just adding some clarity and using some language regarding downloading samples from the multiplex section.